### PR TITLE
isso: fix build, switch to python 3, add a test

### PR DESCRIPTION
**Pull request by fgaz**
###### Motivation for this change

fixing my packages before the 20.09 branch-off
edit: the branch-off happened
ZHF: #97479

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/isso/default.nix [hunk 1]
```diff
@@ -12,15 +16,56 @@ with python2.pkgs; buildPythonApplication rec {
...
+      sha256 = "0pwws93ginlf8scnaawqarhbf8dklx1rjsa4kna68bq3lp84bh70";
+    })
+
+    # https://github.com/posativ/isso/issues/659
+    # https://github.com/posativ/isso/pull/660
+    (fetchpatch {
+      name = "flask-caching-fix";
+      url = "https://github.com/posativ/isso/pull/660.patch";
+      sha256 = "09jgd28ldf2zhmg83fjwrimfn97pvcd5fnssi4241gd7z8dafi58";
+    })
```
Comment (jonringer) [inline pkgs/servers/isso/default.nix:49]: at this point, should we just bump to unstable?

Comment (fgaz) [reply pkgs/servers/isso/default.nix:49]: Of course if you prefer we can bump to unstable

**Comment by Enteee** [comment]
#97717 was merged, I think this can be closed.

**Pull request closed without merging**